### ISSUE TITLE
Nombre de tablero inconsistente

### ISSUE THREAD

**Issue by luisrovirosa** [tmoreno/jbrains-trivia#4]
Me ha resultado raro ver [tablero](https://github.com/tmoreno/jbrains-trivia/blob/master/game.js#L10) y que en realidad fuera a qué [categoría corresponde](https://github.com/tmoreno/jbrains-trivia/blob/master/start.js#L10).

**Comment by tmoreno** [comment]
Sí, quizás tenga un nombre desafortunado, entiendo que el tablero lo conforman casillas que en realidad son las categorías de las preguntas, como el trivial.
